turbo.json

```diff
@@ -1,39 +1,54 @@
 {
+  "$schema": "https://turbo.build/schema.json",
+  "globalDependencies": ["package.json"],
   "pipeline": {
     "build": {
-      "dependsOn": ["^build", "intl-compile", "codegen"],
+      "dependsOn": ["^build", "build:css", "intl-compile", "codegen"],
       "inputs": ["src/**", "package.json"],
-      "outputs": ["dist/**"]
+      "outputs": ["dist/**"],
+      "outputMode": "new-only"
     },
     "build:css": {
-        "outputs": ["src/assets/css/hydrogen.css"]
+      "inputs": ["src/**"],
+      "outputs": ["src/assets/css/hydrogen.css"]
     },
     "codegen": {
```

Comment:
I tried changing a graphql file and rerunning `npm run dev` and it used cached `codgen` but reran `intl-extract` and `intl-compile`?

Reply:
worked OK this time :shrug: